src/calculateRentalCost.js

```diff
@@ -29,6 +29,20 @@
 
 function calculateRentalCost(days) {
   // write code here
+  const dayTariff = 40;
+  const discount20 = 20;
+  const discount50 = 50;
+  const totalTariff = days * dayTariff;
+
+  if (days >= 3 && days < 7) {
+    return totalTariff - discount20;
+  }
+
+  if (days >= 7) {
+    return totalTariff - discount50;
+  }
```

Comment:
You can simplify this.
Avoid double conditions.
Switch these blocks and use single condition inside if statement.

Reply:
@VinnieJ-2k20  Simplified as per your comment.